frontend/views/containers/chatroom/ChatMembersAllModal.vue

```diff
@@ -95,20 +95,22 @@ modal-base-template.has-background(
                   .c-display-name(v-if='displayName' data-test='profileName') @{{ username }}
 
             .c-actions(v-if='isJoined')
-              i18n.button.is-outlined.is-small(
+              button-submit.button.is-outlined.is-small(
                 v-if='!joinedDate'
-                tag='button'
+                type='button'
                 @click.stop='addToChannel(username)'
                 :data-test='"addToChannel-" + username'
-                :args='LTags("span")'
-              ) Add {span_} to channel{_span}
+              )
+                i18n(:args='LTags("span")') Add {span_} to channel{_span}
```

Comment:
wasn't quite sure about why span_` and `_span` are used here but just left it as is for now.